autogpt_platform/backend/backend/data/graph.py

```diff
@@ -51,17 +51,8 @@ class Node(BaseDbModel):
     block_id: str
     input_default: BlockInput = {}  # dict[input_name, default_value]
     metadata: dict[str, Any] = {}
-
```

Comment:
Removed private field since it needs to be propagated through an RPC.

Reply:
Do you mean this break with `@expose` otherwise?

Reply:
With expose input_links and output_links are not propagated. we are not calling `from_db` method after all. It's serialized and deserialized.

Reply:
As long as they are included in `model_dump()` it should work, right? Are `_` prefixed fields not dumped?

Reply:
Ohh I see the `PrivateAttr` now :')